images/win/scripts/Installers/Validate-Toolset.ps1

```diff
@@ -11,11 +11,15 @@ function Run-ExecutableTests {
     )
 
     foreach ($executable in $Executables) {
+        $versionCommand = "--version"
+        If ($executable -Match "go.exe") {
+            $versionCommand = "version"
+        }

```

Comment:
Is there any more generic solution here?

Reply:
Possibly we can add check command field in toolset.json ?

Reply:
`If ($executable -Match "go.exe") {` replace to check for tool name instead of binary name

Reply:
Done.